bootstrap/gce/util.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Use the config file specified in $APOLLO_CONFIG_FILE, or default to
+# config-default.sh.
+APOLLO_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${APOLLO_ROOT}/bootstrap/${APOLLO_PROVIDER}/${APOLLO_CONFIG_FILE-"config-default.sh"}"
+
+verify_prereqs() {
+  if [[ "$(which terraform)" == "" ]]; then
+    echo -e "${color_red}Can't find terraform in PATH, please fix and retry.${color_norm}"
+    exit 1
+  fi
+
+  check_terraform_version
+
+  if [[ "$(which ansible-playbook)" == "" ]]; then
+    echo -e "${color_red}Can't find ansible-playbook in PATH, please fix and retry.${color_norm}"
+    exit 1
+  fi
+  if [[ "$(which python)" == "" ]]; then
+    echo -e "${color_red}Can't find python in PATH, please fix and retry.${color_norm}"
+    exit 1
+  fi
+  if [[ "$(pip list | grep boto)" == "" ]]; then
```

Comment:
Actually any provider need it now.
Those scripts would be removed/refactor with https://github.com/Capgemini/Apollo/pull/290